addons/project/static/src/components/subtask_one2many_field/task_view_button.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module */
+
+import { ViewButton } from "@web/views/view_button/view_button";
+import { useService } from "@web/core/utils/hooks";
+
+export class TaskViewButton extends ViewButton {
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+    }
+    async onClick(ev) {
+        const record = this.props.record;
+        if (this.props.clickParams.name != "action_open_task") {
+            super.onClick(ev);
+        }
+        if (await record.save({ noReload: true })) {
+            const action = await this.orm.call(this.props.record.resModel, "action_open_task", [this.props.record.data.id]);
+            this.action.doAction(action);
+        }
+    }
+}
+
+TaskViewButton.props = [...ViewButton.props]
```

Comment:
This can be right after class declaration
```
export class TaskViewButton extends ViewButton {
    static props = ViewButton.props;
```
But is it necessary? As is extends `ViewButton`, it should inherit its props, right?